SmallCommitsWorkshopTests/Controllers/FizzBuzzApiTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.Extensions.DependencyInjection;
+using NUnit.Framework;
+using SmallCommitsWorkshop;
+using SmallCommitsWorkshop.Models;
+
+namespace SmallCommitsWorkshopTests.Controllers {
+	[TestFixture]
+	internal sealed class FizzBuzzApiTests {

```

Comment:
Should we also rename `UserControllerTests` to `UserApiTests` in `master`? I was in experimental phase and did not think we would be using any of the FizzBuzz code.

Reply:
I just needed a name to separate the integration tests from the unit tests.

Reply:
Maybe I should go with e.g. `FizzBuzzControllerIntegrationTests` and `FizzBuzzControllerUnitTests`?

Reply:
That sounds good. I'm game for w/e as long as they're consistent and separated like you said.

Reply:
I've renamed to `FooUnitTests` and `BarTests`, with the later being implicitly integration tests.  This is purposely awkward while I experiment with how this impacts the diff of the big change.  E.g. `FizzBuzzControllerTests` now appears to be _modified_ instead of _moved_.  Not sure I'll keep this long term, but I'm trying it out for now.

Reply:
I'm game if you want to do the rename change and move that into `master` in a separate PR.

Reply:
I like having the `master` state be "wrong" or "undesirable" in some ways.  It's more like real life.  I'm waffling on which "wrong" things are good/acceptable.  I didn't like leaving undisposed disposables because they're easy to miss and lead to runtime errors; in contrast, poorly/inaccurately named tests aren't directly dangerous.